From 75dc131f0575cbe5d885af8897cbde054f7c07cf Mon Sep 17 00:00:00 2001
Message-Id: <75dc131f0575cbe5d885af8897cbde054f7c07cf.1576197846.git.liu.denton@gmail.com>
In-Reply-To: <cover.1576197846.git.liu.denton@gmail.com>
References: <da9321b1bd56aafd16c8dcb99d5d628b79e2244e.1576100147.git.liu.denton@gmail.com>
	<cover.1576197846.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Wed, 11 Dec 2019 14:13:34 -0800
Subject: [PATCH v2 1/2] gitk: drop quotes in copysummary format
To: Git Mailing List <git@vger.kernel.org>,
    Paul Mackerras <paulus@ozlabs.org>
Cc: Beat Bolli <dev+git@drbeat.li>

In an earlier commit[1], git learned the 'reference' pretty format. This
format was very similar to the format that copysummary used but it omits
the quotes surrounding the commit's subject. Remove the quotes from the
format in copysummary in order to match the 'reference' pretty format.

It seems the consensus is that the unquoted form is used more often
anyway[2] so this change should be acceptable.

[These commits are in git.git.]
[1]: 1f0fc1db85 (pretty: implement 'reference' format, 2019-11-19)
[2]: fb2ffa77a6 (SubmittingPatches: remove dq from commit reference, 2019-11-19)

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 gitk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gitk b/gitk
index abe4805ade..d07e3302de 100755
--- a/gitk
+++ b/gitk
@@ -9429,7 +9429,7 @@ proc mktaggo {} {
 proc copysummary {} {
     global rowmenuid autosellen
 
-    set format "%h (\"%s\", %ad)"
+    set format "%h (%s, %ad)"
     set cmd [list git show -s --pretty=format:$format --date=short]
     if {$autosellen < 40} {
         lappend cmd --abbrev=$autosellen
-- 
2.24.1.664.g198078bb5a

